controllers/storageclaim_controller.go

```diff
@@ -207,6 +219,20 @@ func (r *StorageClaimReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	return result, nil
 }
 
+func (r *StorageClaimReconciler) mapCRDAvailability(crdNames ...string) map[string]bool {
```

Comment:
Do we need to make crdNames as input? instead of that can we have it as var inside this method only?

Reply:
What do you mean? We need to be able to send to this function all the CRD names that need to go through this flow path.

Reply:
OK now that I converted it to a util function it might make more sense to you